server/sonar-server/src/main/java/org/sonar/server/issue/filter/IssueFilterService.java

```diff
@@ -223,19 +227,19 @@ public String getLoggedLogin(UserSession userSession) {
   }
 
   public void verifyCurrentUserCanReadFilter(DefaultIssueFilter issueFilter, String login) {
-    if (!issueFilter.user().equals(login) && !issueFilter.shared()) {
+    if (issueFilter.user() != null && !issueFilter.user().equals(login) && !issueFilter.shared()) {
```

Comment:
It's not the same class... But I can have another method ``isFilterOwnedByLogin`` which takes a ``DefaultIssueFilter`` instead of an ``IssueFilterDto``